ros_buildfarm/wrapper/apt_get.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
```

Comment:
None of the other source files in this repo has any so I wouldn't add it for these two files which have only been moved in this PR.